8_bit_pic.py

```diff
@@ -11,102 +37,352 @@
 logger = logging.getLogger(__name__)
 
 WORD_SIZE = 8
+FLASH_SIZE = 2 * 1024 * 1024
+RAM_SIZE = 4 * 1024
```

Comment:
Using the same memory space for code and data - 2 MB flash, followed by 4 kB RAM.